pyomo/contrib/cp/interval_var.py

```diff
@@ -80,7 +82,9 @@ class IntervalVarPresence(ScalarBooleanVar):
 
     __slots__ = ()
 
-    def __init__(self):
+    def __init__(self, *args, **kwd):
+        # TODO: adding args and kwd above made Reference work, but we
+        # probably shouldn't just swallow them, right?
         super().__init__(ctype=IntervalVarPresence)
```

Comment:
@jsiirola, I made this change to make `Reference(interval_var[:].is_present)` work, but I don't really understand what I did or know if this is the right fix.